### PR TITLE
coverity 1021720: Uninitialized scalar field

### PR DESCRIPTION
**Pull request by vmamidi**

### CODE REVIEW

File: plugins/esi/lib/EsiParser.cc [hunk 1]
```diff
@@ -56,7 +56,7 @@ const EsiParser::EsiNodeInfo EsiParser::ESI_NODES[] = {
 const EsiParser::EsiNodeInfo EsiParser::HTML_COMMENT_NODE_INFO(DocNode::TYPE_HTML_COMMENT, "<!--esi", 7, "-->", 3);
 
 EsiParser::EsiParser(const char *debug_tag, ComponentBase::Debug debug_func, ComponentBase::Error error_func)
-  : ComponentBase(debug_tag, debug_func, error_func), _parse_start_pos(-1)
+  : ComponentBase(debug_tag, debug_func, error_func), _parse_start_pos(-1), _orig_output_list_size(0)
```
Comment (PSUdaemon) [inline plugins/esi/lib/EsiParser.cc]: Declaration initializers?

**Comment by atsci** [comment]
RAT check *successful*! https://ci.trafficserver.apache.org/job/RAT-github/519/

**Comment by shukitchan** [comment]
i think it is fine.

File: plugins/esi/lib/EsiParser.cc (hunk 1 above)
Comment (vmamidi) [inline plugins/esi/lib/EsiParser.cc]: changed in PR #1862

**Comment by atsci** [comment]
clang format *successful*! https://ci.trafficserver.apache.org/job/clang-format-github/510/

**Comment by atsci** [comment]
AU check *successful*! https://ci.trafficserver.apache.org/job/autest-github/506/

**Comment by atsci** [comment]
Intel CC build *successful*! https://ci.trafficserver.apache.org/job/icc-github/635/

**Comment by atsci** [comment]
FreeBSD11 build *successful*! https://ci.trafficserver.apache.org/job/freebsd-github/2202/

**Comment by atsci** [comment]
Linux build *successful*! https://ci.trafficserver.apache.org/job/linux-github/2098/

**Comment by atsci** [comment]
clang-analyzer build *successful*! https://ci.trafficserver.apache.org/job/clang-analyzer-github/763/

**Pull request closed without merging**